### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Frenkiee**

### CODE REVIEW

File: api/src/controllers/contract.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const accountTokenBalance = async (req: AppRequest<TokenBalanceParam>, re
       const token = await findERC20Token(req.body.contractAddress);
       res.send({ balance: 0, decimals: token.decimals });
     } else {
-      res.send({ balance: tokenBalances[0].balance, decimals: tokenBalances[0].decimals });
+      res.send({ balance: tokenBalances[0].balance, decimals: tokenBalances[0].info.decimals || 18 });
```
Comment (frisitano) [inline api/src/controllers/contract.ts]: why is decimals defaulted to 18?

**Comment by frisitano** [comment]
`ILIKE` pattern is reasonable for a temp solution but should not be left long term as it does not leverage indexes and is therefore inefficient.  We should identify source of case inconsistency and fix at source.

File: api/src/controllers/contract.ts (hunk 1 above)
Comment (Frenkiee) [reply api/src/controllers/contract.ts]: nfts do not have decimals, I've changed it to 0

**Comment by Netherdrake** [comment]
`toContractAddress` should really be called `toChecksumAddress`. Also, it should be applied to accounts and not just contracts.

**Pull request merged**